### PR TITLE
Update SemanticDB version to 4.2.5 and enable previously broken test

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

**Comment by olafurpg** [comment]
The test failure for `unit/test` is legitimate, you can fix it by running `sbt save-expect`

**Comment by olafurpg** [comment]
Looks like the 4.2.4 release didn't complete the full cross-build, I have triggered a new v4.2.5 release.

**Comment by tgodzik** [comment]
> Looks like the 4.2.4 release didn't complete the full cross-build, I have triggered a new v4.2.5 release.

Will try with the new one, I haven't had time to take a look at it, but will make sure everything is working with 4.2.5 on Monday

**Comment by olafurpg** [comment]
The release job is running at https://travis-ci.org/scalameta/scalameta/jobs/606391878 🍿 let's hope it completes before the Travis timeout. Might be a good idea to migrate the Scalameta release job to GitHub Actions to avoid the timeouts

**Comment by olafurpg** [comment]
Also probably time to upgrade the scalameta build to sbt v1.3 so that we can use the latest sbt-sonatype to speed up the publish step

**Comment by olafurpg** [comment]
Release succeeded, artifacts should be on Maven in the next ~30 minutes

**Comment by olafurpg** [comment]
Release is up now on Maven, all artifacts seem to have uploaded successfully

**Comment by tgodzik** [comment]
Test failures seem legit, will take a look on Monday.

**Comment by tgodzik** [comment]
@olafurpg Also I think 2.12.7 is not published, but I think we should most likely drop it and deprecate 2.12.8.

**Comment by olafurpg** [comment]
I'm in favor of dropping 2.12.7 and deprecating 2.12.8/2.12.9

File: tests/slow/src/test/scala/tests/gradle/GradleLspSuite.scala [hunk 1]
```diff
@@ -284,7 +284,7 @@ object GradleLspSuite extends BaseImportSuite("gradle-import") {
       _ = assertNoDiff(
         client.messageRequests.peekLast(),
         // only 3 projects since no empty test targets are created for gradle
```
Comment (marek1840) [inline tests/slow/src/test/scala/tests/gradle/GradleLspSuite.scala]: > only 3

now it's 5 ;)

Comment (tgodzik) [reply tests/slow/src/test/scala/tests/gradle/GradleLspSuite.scala]: Updated the comment not to include a number - that was not important really.

**Pull request merged**